WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -1425,6 +1426,11 @@ public static Intent createMainActivityAndShowBloggingPromptsOnboardingActivityI
         return intent;
     }
 
+    public static void showBloggingPromptsListActivity(final Activity activity) {
+        Intent intent = BloggingPromptsListActivity.createIntent(activity);
```

Comment:
Nitpick: would it make sense to make it `final`?

Reply:
I agree! Changed!